test/e2e/run_test.sh

```diff
@@ -78,11 +78,34 @@ function setup_istio() {
     kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
         { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v1.1.0" | kubectl apply -f -; }
 
-    istioctl install --set profile=ambient --skip-confirmation
+    istioctl install --set profile=ambient --set meshConfig.accessLogFile="/dev/stdout" --skip-confirmation
 }
 
 function setup_kmesh() {
     helm install kmesh $ROOT_DIR/deploy/helm -n kmesh-system --create-namespace --set deploy.kmesh.image.repository=localhost:5000/kmesh
+
+    # Wait for all Kmesh pods to be ready.
+    while true; do
```

Comment:
nit: we can abstract a function like this

https://github.com/kurator-dev/kurator/blob/main/hack/util.sh#L105-L120